public/index.html

```diff
@@ -6,8 +6,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta
-      name="description"
-      content="Web site created using create-react-app"
+      name="GDSC Lead Map"
```

Comment:
```suggestion
      name="description"
```

Reply:
Shouldn't the name be GDSC Lead Map?

Reply:
@abhinavsri360 He changed the `meta` property name. `GDSC Lead Map` is not a valid `meta` property.

Reply:
Ohh, I didn't know that before, sorry 👀

Reply:
Oops, my bad :P
I'll submit a clean PR